src/components/diagnosticReportView/GeneralView.tsx

```diff
@@ -68,37 +84,57 @@ const GeneralView: React.FC<GeneralViewProps> = ({ id, generalViews, setGeneralV
         setOptions={setGeneralViews}
       />
       <div className={classes.imageContainer}>
-        {generalViewSelections.map((option) => (
-          <img
-            key={option.finalPath}
-            className={classes.image}
-            src={reportUrl(option.finalPath, id)}
-            alt={option.finalPath}
-            onError={(e: React.SyntheticEvent<HTMLImageElement, Event>) => {
-              let newUrl;
+        {generalViewSelections.map((option) => {
```

Comment:
How is the ordering of the plots determined?

Reply:
I think it is the order of the selection